### ISSUE TITLE
Service is not set to enabled on boot with default role settings

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-varnish#67]
After using the defaults and deploying to an Ubuntu 16.04 instance, I'm seeing:

```
# systemctl status varnish
...
/etc/systemd/system/varnish.service; disabled
```

And if I reboot, Varnish is not started after boot... therefore I have to manually start it. Maybe this is an issue with the `service` module and systemd, and the Varnish repo?

**Comment by geerlingguy** [comment]
Ah... I think I hit https://github.com/geerlingguy/ansible-role-varnish/pull/65/files — and that was fixed in 2.0.2. I should upgrade this project from 2.0.0, which still had that problem.

**Comment by geerlingguy** [comment]
Yep.
